documentation/book/appendix_metrics.adoc

```diff
@@ -52,19 +52,33 @@ The following resources are defined:
 
 ==== Deploying on {OpenShiftName}
 
-To deploy all these resources you can run the following.  Note that this file creates a `ClusterRoleBinding` in the `myproject` namespace.  If you're not using this namespace then download the resource file locally and update it.
+The provided YAML file, with all the Prometheus related resources, creates a `ClusterRoleBinding` in the `myproject` namespace.
+If you're not using this namespace then download the resource file locally and update it as follow.
```

Comment:
If you are using a different namespace, download the resource file and update it as follows: